ddl/index.go

```diff
@@ -614,15 +685,27 @@ func (w *addIndexWorker) backfillIndexInTxn(handleRange reorgIndexTask) (nextHan
 			return errors.Trace(err)
 		}
 
+		err = w.batchCheckUniqueKey(txn, idxRecords)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
 		for _, idxRecord := range idxRecords {
+			// The index is already exists, we skip it, no needs to backfill it.
+			// The following update, delete, insert on these rows, TiDB can handle it correctly.
+			if idxRecord.skip {
+				continue
```

Comment:
Is skip maybe cause the `addedCount ` wrong?  see this PR : https://github.com/pingcap/tidb/pull/6980/files

Reply:
No, this skipped row will not affect scanCount or addedCount, it is expected